cloudformation.yaml

```diff
@@ -91,6 +78,13 @@ Resources:
           - kms:DescribeKey
           Resource: "*"
 
+  LambdaKMSKeyAlias:
+    Type: AWS::KMS::Alias
+    Properties:
+      AliasName: !Sub alias/zuora-crediter-kms
+      TargetKeyId: !Ref LambdaKMSKey
+    DependsOn: LambdaKMSKey
+
   Lambda:
     Type: AWS::Lambda::Function
     Properties:
```

Comment:
I know you haven't changed this section as part of this PR, but setting your environment variables = empty strings will overwrite your details whenever this stack is updated by CloudFormation.